backend/locust/cleanup_experiments.py

```diff
@@ -0,0 +1,15 @@
+from  upgrade_mathia_data import modules, workspaces
+import deleteExperiment
+
+protocol = "http"
+host = "localhost:3030"
+
+option = int(input("Enter the number of  experiments to delete: "))
```

Comment:
This file should be improved. Dangerous to use even in QA. Perhaps an interactive prompt to create/delete experiments

Reply:
Splitting it out from main script was appropriate. Create a separate task to improve create/delete process.